### DISCUSSION TITLE
How to deploy with Cloudflate

### DISCUSSION THREAD

**Discussion by KanekiHits** [UseInterstellar/Interstellar#300] [Q&A] [answered]
I like the pages.dev, and i have my own custom domain on there and would like to deploy interstellar on there, could somebody help i've seen people deploy on cloud flare before

**Comment by Hamidcc** [comment]
im pretty sure you cant upload interstellar there, try reading the file

**Comment by xbubbo** [comment, marked as answer]
Bro skidded my site
